controllers/articles-controller.js

```diff
@@ -1,4 +1,4 @@
-const { selectTopics } = require("../models/articles-model")
```

Comment:
You should look at organising your models and controllers based on the data they are dealing with. 
For example, topics could have it's own model/controller and articles could have it's own model/controller.